<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NameToolsClass</class>
 <widget class="QMainWindow" name="NameToolsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NameTools</string>
  </property>
  <widget class="QWidget" name="Widget">
   <widget class="QLineEdit" name="headname">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>31</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>person</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>31</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文件前缀名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>开始于</string>
    </property>
   </widget>
   <widget class="QListWidget" name="des_list">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>191</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="source_list">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>191</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>原名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>新名称</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始命名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>标识</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numstart_val">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numstart_val_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IRR/VIS</string>
    </property>
   </widget>
   <widget class="QLabel" name="source_add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>195</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="des_add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>440</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Version   1.0.1(x64build)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>462</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Offical Release Jan 13,2019</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>168</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>源目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="source_btn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="numhead">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>40</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数字前缀</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="NameTools.qrc"/>
 </resources>
 <connections/>
</ui>
